<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneratorConfigWidget</class>
 <widget class="QWidget" name="GeneratorConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>342</width>
    <height>1427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>322</width>
        <height>1407</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="factoryGroup">
         <property name="title">
          <string>Grid Factory</string>
         </property>
         <layout class="QVBoxLayout" name="factoryLayout">
          <item>
           <layout class="QHBoxLayout" name="factorySizeLayout">
            <item>
             <widget class="QLabel" name="sizeLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Size:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="sizeXSpinBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>x</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="sizeYSpinBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="factoryOriginLayout">
            <item>
             <widget class="QLabel" name="originLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Origin:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="originXSpinBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>,</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="originYSpinBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="factorySpacingLayout">
            <item>
             <widget class="QLabel" name="spacingLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Spacing:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="spacingXSpinBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>,</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="spacingYSpinBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="outputGroup">
         <property name="title">
          <string>Output</string>
         </property>
         <layout class="QHBoxLayout" name="outputLayout">
          <item>
           <widget class="QLabel" name="outputFileLabel">
            <property name="text">
             <string>Output File:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="outputFileEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="browseButton">
            <property name="text">
             <string>Browse</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="fillerGroup">
         <property name="title">
          <string>Fillers</string>
         </property>
         <layout class="QVBoxLayout" name="fillerLayout">
          <item>
           <widget class="QListWidget" name="fillerListWidget"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="fillerButtonsLayout">
            <item>
             <widget class="QPushButton" name="addFillerButton">
              <property name="text">
               <string>Add</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="editFillerButton">
              <property name="text">
               <string>Edit</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="removeFillerButton">
              <property name="text">
               <string>Remove</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="saverGroup">
         <property name="title">
          <string>Saver</string>
         </property>
         <layout class="QFormLayout" name="saverLayout">
          <item row="0" column="0">
           <widget class="QLineEdit" name="saverPathEdit"/>
          </item>
          <item row="2" column="0">
           <widget class="QLineEdit" name="paramsLineEdit"/>
          </item>
          <item row="3" column="0">
           <widget class="QLineEdit" name="normsLineEdit"/>
          </item>
          <item row="1" column="0">
           <widget class="QSpinBox" name="stepsForSaveSpinBox"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="materialGroup">
         <property name="title">
          <string>Material</string>
         </property>
         <layout class="QFormLayout" name="materialLayout">
          <item row="0" column="0">
           <widget class="QDoubleSpinBox" name="c1SpinBox"/>
          </item>
          <item row="1" column="0">
           <widget class="QDoubleSpinBox" name="c2SpinBox"/>
          </item>
          <item row="2" column="0">
           <widget class="QDoubleSpinBox" name="rhoSpinBox"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gridGroup">
         <property name="title">
          <string>Grid</string>
         </property>
         <layout class="QFormLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLineEdit" name="gridIdEdit"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="correctorGroup">
         <property name="title">
          <string>Correctors</string>
         </property>
         <layout class="QVBoxLayout" name="correctorLayout">
          <item>
           <widget class="QListWidget" name="correctorListWidget"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="correctorButtonsLayout">
            <item>
             <widget class="QPushButton" name="addCorrectorButton">
              <property name="text">
               <string>Add</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="editCorrectorButton">
              <property name="text">
               <string>Edit</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="removeCorrectorButton">
              <property name="text">
               <string>Remove</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="globalGroup">
         <property name="title">
          <string>Global Settings</string>
         </property>
         <layout class="QFormLayout" name="globalLayout">
          <item row="0" column="0">
           <widget class="QDoubleSpinBox" name="dtSpinBox">
            <property name="prefix">
             <string/>
            </property>
            <property name="suffix">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QSpinBox" name="stepsSpinBox"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="schemaGroup">
         <property name="title">
          <string>Schema</string>
         </property>
         <layout class="QFormLayout" name="schemaLayout">
          <item row="0" column="0">
           <widget class="QLineEdit" name="schemaNameEdit"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="generateButton">
         <property name="text">
          <string>Generate Config File</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
